packages/webapp/src/containers/Task/TaskReadOnly/index.jsx

```diff
@@ -128,11 +128,10 @@ function TaskReadOnly({ history, match, location }) {
     files.push({ url: externalIrrigationPrescription.prescription.vriData.file_url });
   }
   if (task.documents?.length) {
-    files.push(
-      ...task.documents.reduce((acc, cv) => {
```

Comment:
I don't quite understand why... but this caused wacky behaviour:

Going to the readonly completed task and refreshing once or twice would result in the complete task form submission then the task would be overwritten missing the complete_date details.